### DISCUSSION TITLE
Failed to check remote checksum: unknown url type: 'u.md5' ??

### DISCUSSION THREAD

**Discussion by Crash0v3r1de** [Security-Onion-Solutions/securityonion#5244] [General]
So I have my SecurityOnion node in my homelab to watch my lab traffic just to be safe. I have not made any changes to the ruleset except adding some additional rule URLS but I tested the update with those completely removed and that changed nothing.

Is there a known bug for SecurityOnion right now for this type of rule update failure? This box sits and just watches so I have no clue why this randomly stopped working.

> 2021-08-22 03:34:30,671 - <INFO> - Loading ./rulecat.conf.
> 2021-08-22 03:34:30,694 - <INFO> - Forcing Suricata version to 6.0.
> 2021-08-22 03:34:30,694 - <INFO> - Fetching /.
> Traceback (most recent call last):
>   File "/usr/local/bin/idstools-rulecat", line 12, in <module>
>     sys.exit(main())
>   File "/usr/local/lib/python3.9/site-packages/idstools/scripts/rulecat.py", line 845, in main
>     files = Fetch(args).run()
>   File "/usr/local/lib/python3.9/site-packages/idstools/scripts/rulecat.py", line 329, in run
>     files.update(self.fetch(url))
>   File "/usr/local/lib/python3.9/site-packages/idstools/scripts/rulecat.py", line 317, in fetch
>     idstools.net.get(
>   File "/usr/local/lib/python3.9/site-packages/idstools/net.py", line 55, in get
>     remote = urlopen(url)
>   File "/usr/local/lib/python3.9/urllib/request.py", line 214, in urlopen
>     return opener.open(url, data, timeout)
>   File "/usr/local/lib/python3.9/urllib/request.py", line 501, in open
>     req = Request(fullurl, data)
>   File "/usr/local/lib/python3.9/urllib/request.py", line 320, in __init__
>     self.full_url = url
>   File "/usr/local/lib/python3.9/urllib/request.py", line 346, in full_url
>     self._parse()
>   File "/usr/local/lib/python3.9/urllib/request.py", line 375, in _parse
>     raise ValueError("unknown url type: %r" % self.full_url)
> ValueError: unknown url type: '/'

**Comment by weslambert** [comment]
Could you please provide some more information as to how/where you modified the rule URLs?
